cpp/mrc/CMakeLists.txt

```diff
@@ -191,7 +191,7 @@ target_link_libraries(libmrc
     gRPC::grpc++
     gRPC::grpc
     gRPC::gpr
-    ucxx::ucxx
+    ucxx
```

Comment:
Did we figure out why we needed `ucxx` here and not `ucxx::ucxx`? Can we switch this back to `ucxx::ucxx`?

Reply:
Oh yes, good reminder `ucxx::ucxx` works now, I've reverted to the original via https://github.com/nv-morpheus/MRC/pull/437/commits/9d597bf0df8d3e52d7aa117c598727a42c627fd3 .